<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BCExtra</class>
 <widget class="QDialog" name="BCExtra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>709</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="sub_zone_label">
        <property name="text">
         <string>zone name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="zone_name_edit">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="boundary_type_label">
        <property name="text">
         <string>BC name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="BC_name_edit">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="boundary_type_label_2">
        <property name="text">
         <string>BC type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="BC_type_edit">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="angular_velocity_label">
        <property name="text">
         <string>Zone angular velocity:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="angular_velocity_edit">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="no_extra_label">
      <property name="text">
       <string>No extra Boundary Condition need to setup</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="in_out_layout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="velocity_u_lineEdit">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="velocity_u_label">
        <property name="text">
         <string>relative velocity u</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="velocity_profile_num_in_edit">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="velocity_w_label">
        <property name="text">
         <string>relative velocity w</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="profile_distribution_in_combo">
        <item>
         <property name="text">
          <string>x</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>z</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>radius</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="profile_distribution_in_label">
        <property name="text">
         <string>Profile Distribution </string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="velocity_profile_num_in_label">
        <property name="text">
         <string>Boundary Profile Num</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="velocity_w_lineEdit">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="velocity_v_label">
        <property name="text">
         <string>relative velocity v</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="velocity_v_lineEdit">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="velocity_u_unit_label">
        <property name="text">
         <string>m/s</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="velocity_v_unit_label">
        <property name="text">
         <string>m/s</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="velocity_w_unit_label">
        <property name="text">
         <string>m/s</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="visc_wall_layout">
      <item row="1" column="0">
       <widget class="QLabel" name="wall_velocity_u_label">
        <property name="text">
         <string>relative velocity u</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="wall_angular_velocity_label">
        <property name="text">
         <string>relative angular velocity</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="wall_velocity_u_lineEdit">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="wall_temperature_lineEdit">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="wall_velocity_w_lineEdit">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="wall_temperature_label">
        <property name="text">
         <string>wall temperature</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="wall_velocity_v_lineEdit">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="wall_velocity_w_label">
        <property name="text">
         <string>relative velocity w</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="wall_velocity_v_label">
        <property name="text">
         <string>relative velocity v</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="wall_angular_velocity_lineEdit">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="wall_angular_velocity_unit_label">
        <property name="text">
         <string>rad/s</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="wall_velocity_u_unit_label">
        <property name="text">
         <string>m/s</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="wall_velocity_v_unit_label">
        <property name="text">
         <string>m/s</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="wall_velocity_w_unit_label">
        <property name="text">
         <string>m/s</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="wall_velocity_k_unit_label">
        <property name="text">
         <string>K</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="profile_label">
        <property name="text">
         <string>Boundary Profile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="load_profile_button">
        <property name="text">
         <string>Load Profile</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5"/>
    </item>
    <item>
     <widget class="QTableWidget" name="profile_table"/>
    </item>
    <item>
     <widget class="QPushButton" name="clostbutton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
